pkgs/top-level/python-packages.nix

```diff
@@ -29044,6 +29044,24 @@ EOF
     };
   };
 
+  first = buildPythonPackage rec {
```

Comment:
We have a new policy for python to put python packages into a separate file and only reference them here:

See also:

```
$ chromium "file://$(nix-build --no-out-link doc)//share/doc/nixpkgs/manual.html#contributing-guidelines"
```